src/electos/ballotmaker/data/extractor.py

```diff
@@ -0,0 +1,311 @@
+from typing import Dict, List, Union
+
+from electos.ballotmaker.data.models import ElectionData
+from electos.datamodels.nist.indexes import ElementIndex
+from electos.datamodels.nist.models.edf import (
+    BallotMeasureContest,
+    BallotMeasureSelection,
+    BallotStyle,
+    Candidate,
+    CandidateContest,
+    CandidateSelection,
+    Election,
+    ElectionReport,
+    InternationalizedText,
+    OrderedContest,
+    OrderedHeader,
+)
+
+
+# --- Base Types
+#
+# Schema expresses these as union types not subclasses
+
+Contest = Union[BallotMeasureContest, CandidateContest]
+
+OrderedContent = Union[OrderedContest, OrderedHeader]
+
+
+# --- Utilities
+
+
+def _text_content(item):
+    """Return joined lines from internationalized text."""
+    assert isinstance(item, InternationalizedText)
```

Comment:
Yes, I'm aware of it. It's an old check from the early revisions of this branch. I deferred on fixing it because the right answer is to use type annotations, and I was out of time. I have these coming in a future change.

Reply:
Sounds good.